frontend/src/features/admin-form/settings/SettingsEmailsPage.tsx

```diff
@@ -1,49 +1,76 @@
-import { FormControl, Skeleton } from '@chakra-ui/react'
+import { Skeleton } from '@chakra-ui/react'
 
-import { FormResponseMode } from '~shared/types'
+import { FormResponseMode, FormSettings, FormStatus } from '~shared/types/form'
+import { PaymentChannel } from '~shared/types/payment'
 
 import FormLabel from '~components/FormControl/FormLabel'
 import { TagInput } from '~components/TagInput'
 
 import { CategoryHeader } from './components/CategoryHeader'
-import { EmailFormSection } from './components/EmailFormSection'
+import { EmailNotificationsHeader } from './components/EmailNotificationsHeader'
+import { FormEmailSection } from './components/FormEmailSection'
+import { MrfFormEmailSection } from './components/MrfFormEmailSection'
 import { useAdminFormSettings } from './queries'
 
-const AdminEmailSection = () => {
-  const { data: settings } = useAdminFormSettings()
-
-  if (!settings) {
-    return <EmailFormSectionSkeleton />
-  }
+const FormEmailSectionSkeleton = (): JSX.Element => {
+  return (
+    <Skeleton>
+      <FormLabel>Send an email copy of new responses</FormLabel>
+      <TagInput placeholder="me@example.com" isDisabled />
+    </Skeleton>
+  )
+}
 
-  const isEmailOrStorageMode =
-    settings?.responseMode === FormResponseMode.Email ||
-    settings?.responseMode === FormResponseMode.Encrypt
+interface FormEmailSectionContainerProps {
+  isDisabled: boolean
+  settings: FormSettings
+}
 
-  // should render null
-  if (!isEmailOrStorageMode) {
-    return null
+const FormEmailSectionContainer = ({
+  isDisabled,
+  settings,
+}: FormEmailSectionContainerProps): JSX.Element => {
+  if (settings.responseMode === FormResponseMode.Multirespondent) {
+    return <MrfFormEmailSection isDisabled={isDisabled} settings={settings} />
+  } else {
```

Comment:
/nit Don't an else here. 

P.S. I generally favor guard clauses and to avoid using and else. Keeps me from creating messy deeply nested if-else.